js/components/main.js

```diff
@@ -822,19 +822,17 @@ class Main extends ImmutableComponent {
   }
 
   get customTitlebar () {
-    const customTitlebarEnabled = isWindows
+    const customTitlebarEnabled = true // isWindows TODO revert before merge
     const captionButtonsVisible = customTitlebarEnabled
     const menubarVisible = customTitlebarEnabled && (!getSetting(settings.AUTO_HIDE_MENU) || this.props.windowState.getIn(['ui', 'menubar', 'isVisible']))
-    const selectedIndex = this.props.windowState.getIn(['ui', 'menubar', 'selectedIndex'])
+    const tempContextMenuIndex = this.props.windowState.getIn(['ui', 'contextMenu', 'selectedIndex'])
```

Comment:
Regarding the previous comments, it looks like you already had it in here :smile: Any reason you chose to use the version from WindowStore over this one?  It's important to rely on this version, so that the control re-renders when the store changes